src/ext/ca/serverca/scasched/scauser.cpp

```diff
@@ -515,7 +515,7 @@ HRESULT ScaUserExecute(
         if (wzDomain && *wzDomain)
         {
             er = ::DsGetDcNameW(NULL, wzDomain, NULL, NULL, NULL, &pDomainControllerInfo);
-            if (HRESULT_FROM_WIN32(er) == RPC_S_SERVER_UNAVAILABLE)
+            if (RPC_S_SERVER_UNAVAILABLE == er)
```

Comment:
I'm not excited about turning on this code path for the first time ever (since it seems the existing error check should never be true) without a good idea when it should be called. Have you tested your suggestion and found it works correctly?